app/src/main/java/ch/hikemate/app/ui/profile/ProfileScreen.kt

```diff
@@ -0,0 +1,149 @@
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import ch.hikemate.app.R
+import ch.hikemate.app.model.profile.HikingLevel
+import ch.hikemate.app.model.profile.Profile
+import ch.hikemate.app.model.profile.ProfileViewModel
+import ch.hikemate.app.ui.components.BigButton
+import ch.hikemate.app.ui.components.ButtonType
+import ch.hikemate.app.ui.navigation.LIST_TOP_LEVEL_DESTINATIONS
+import ch.hikemate.app.ui.navigation.NavigationActions
+import ch.hikemate.app.ui.navigation.Route
+import ch.hikemate.app.ui.navigation.Screen
+import ch.hikemate.app.ui.navigation.SideBarNavigation
+import com.google.firebase.Timestamp
+
+object ProfileScreen {
+  const val TEST_TAG_TITLE = "profileScreenTitle"
+  const val TEST_TAG_NAME = "profileScreenNameInfo"
+  const val TEST_TAG_EMAIL = "profileScreenEmailInfo"
+  const val TEST_TAG_HIKING_LEVEL = "profileScreenHikingLevelInfo"
+  const val TEST_TAG_JOIN_DATE = "profileScreenJoinDateInfo"
+  const val TEST_TAG_EDIT_PROFILE_BUTTON = "profileScreenEditProfileButton"
+  const val TEST_TAG_SIGN_OUT_BUTTON = "profileScreenSignOutButton"
+}
+
+/**
+ * A composable to display an information of the profile.
+ *
+ * @param label The label of the information.
+ * @param value The value of the information.
+ */
+@Composable
+fun DisplayInfo(label: String, value: String, modifier: Modifier = Modifier) {
+  Column(
+      verticalArrangement = Arrangement.spacedBy(2.dp),
+  ) {
+    Text(label, style = TextStyle(fontWeight = FontWeight.Bold, fontSize = 20.sp))
+    Text(value, style = TextStyle(fontSize = 20.sp), modifier = modifier)
+  }
+}
+
+/**
+ * A composable that displays the profile screen.
+ *
+ * @param navigationActions The navigation actions.
+ * @param profileViewModel The profile view model.
+ */
+@Composable
+fun ProfileScreen(
+    navigationActions: NavigationActions,
+    profileViewModel: ProfileViewModel = viewModel(factory = ProfileViewModel.Factory)
+) {
+  val context = LocalContext.current
+
+  // TODO: show an error if the profile is null. For now display it for test purposes
+  val profileState = profileViewModel.profile.collectAsState()
+
+  val profile: Profile =
+      profileState.value
+          ?: Profile(
+              "custom-id",
+              "John Doe",
+              "john.doe@gmail.com",
+              HikingLevel.INTERMEDIATE,
+              Timestamp.now())
+
+  SideBarNavigation(
+      onTabSelect = { navigationActions.navigateTo(it) },
+      tabList = LIST_TOP_LEVEL_DESTINATIONS,
+      selectedItem = Route.PROFILE) { p ->
+        Column(
+            modifier =
+                Modifier.testTag(Screen.PROFILE)
+                    .padding(p)
+                    .padding(
+                        // Add padding to the sidebar padding
+                        start = 16.dp,
+                        end = 16.dp,
+                        top = 16.dp,
+                    ),
```

Comment:
Nitpick: Either clarify what those multiple paddings do with a better comment or change it to don't have multiple paddings
```suggestion
                    .padding(
                        // Add padding to the sidebar padding
                        start = p.calculateStartPadding(LayoutDirection.Rtl) + 16.dp,
                        end = p.calculateEndPadding(LayoutDirection.Rtl) + 16.dp,
                        top = p.calculateTopPadding() + 16.dp,
                        bottom = p.calculateBottomPadding()
                    ),
```

Reply:
From my opinion (and other from telegram), the first one is cleaner but less clearer. I added a better comment to explain what's happening here.

Reply:
Agreed on Telegram